config/config.go

```diff
@@ -524,9 +527,12 @@ func NewConfig() *Config {
 	flag.Var(cfg.CredentialPaths, "credentials-paths", "directories or files to watch for credentials to use by bearerinjector filter")
 	flag.DurationVar(&cfg.CredentialsUpdateInterval, "credentials-update-interval", 10*time.Minute, "sets the interval to update secrets")
 	flag.BoolVar(&cfg.EnableOpenPolicyAgent, "enable-open-policy-agent", false, "enables Open Policy Agent filters")
+	flag.BoolVar(&cfg.EnableOpenPolicyAgentOverridePeriodTriggers, "enable-open-policy-agent-override-period-triggers", false, "when enabled skipper will set all plugin triggers to manual and trigger all plugins directly, which includes f.ex. to download of new bundles")
```

Comment:
```suggestion
	flag.BoolVar(&cfg.EnableOpenPolicyAgentOverridePeriodTriggers, "enable-open-policy-agent-skipper-controlled-triggers", false, "when enabled, Skipper will use a central control loop trigger plugins, which includes f.ex. to download of new bundles")
```

Reply:
I will leave this open for other opinions on the name of the parameter, as changing it will reflect in quite some test changes.
current options:
1. `enable-open-policy-agent-override-period-triggers`
2. `enable-open-policy-agent-skipper-controlled-triggers`
3. `enable-open-policy-agent-custom-plugin-triggers`

Reply:
the configuration will switch from the opa default behaviour of periodically triggering the bundle to a setup where the opa-filter is triggering the bundles, allowing for better error handling (like retries) and more control (like adding the jitter)

Reply:
I like this one best

Reply:
Lets have it like that then

Reply:
addressed